distmv/normal.go

```diff
@@ -58,6 +58,25 @@ func NewNormal(mu []float64, sigma mat64.Symmetric, src *rand.Rand) (*Normal, bo
 	return n, true
 }
 
+// NewNormalChol creates a new Normal distribution with the given mean and
```

Comment:
I think the wording of this could be improved wrt "and cholesky decomposition of the covariance matrix." Maybe, "and covariance matrix represented by its Cholesky decomposition." or similar.